lib/BiochemistryAPI/BiochemistryAPIImpl.py

```diff
@@ -83,24 +93,26 @@ def get_reactions(self, ctx, params):
         """
         # ctx is the context object
         # return variables are: out_reactions
-        #BEGIN get_reactions
+        # BEGIN get_reactions
         logging.info("Starting get_reactions")
         logging.info("Params: {}".format(params))
-        utils.check_param(params, ['reactions'])
+        utils.check_param(params, ["reactions"])
         out_reactions = []
-        for x in params['reactions']:
-            id = x.split('/')[-1]
+        for x in params["reactions"]:
+            id = x.split("/")[-1]
             rxn = self.reactions.get(id, None)
             if rxn:
-                rxn['aliases'] = self.rxn_aliases.get(id, '')
-                rxn['enzymes'] = self.ec_classes.get(id, '')
+                rxn["aliases"] = self.rxn_aliases.get(id, "")
+                rxn["enzymes"] = self.ec_classes.get(id, "")
             out_reactions.append(rxn)
-        #END get_reactions
+        # END get_reactions
 
         # At some point might do deeper type checking...
         if not isinstance(out_reactions, list):
-            raise ValueError('Method get_reactions return value ' +
-                             'out_reactions is not type list as required.')
+            raise ValueError(
+                "Method get_reactions return value "
+                + "out_reactions is not type list as required."
+            )
         # return the results
         return [out_reactions]
```

Comment:
I _think_ everything after the `#END get_reactions` stub will get dynamically overwritten once `kb-sdk` runs, right? I thought it gets replaced with
```
        #END get_reactions

        # At some point might do deeper type checking...
        if not isinstance(output, dict):
            raise ValueError('Method run_amplicon_workflow return value ' +
                             'output is not type dict as required.')
        # return the results
        return [output]
```

Reply:
Yup, it'd all be overwritten by `kb-sdk`. 😖